auth/api/logging.go

```diff
@@ -82,17 +82,28 @@ func (lm *loggingMiddleware) Identify(ctx context.Context, key string) (id auth.
 	return lm.svc.Identify(ctx, key)
 }
 
-func (lm *loggingMiddleware) Authorize(ctx context.Context, token, sub, obj, act string) (auth bool, err error) {
+func (lm *loggingMiddleware) Authorize(ctx context.Context, subject, object, relation string) (authorized bool, err error) {
 	defer func(begin time.Time) {
-		message := fmt.Sprintf("Method authorize took %s to complete", time.Since(begin))
+		message := fmt.Sprintf("Method authorize %s#%s@%s took %s to complete", object, relation, subject, time.Since(begin))
 		if err != nil {
 			lm.logger.Warn(fmt.Sprintf("%s with error: %s.", message, err))
 			return
 		}
 		lm.logger.Info(fmt.Sprintf("%s without errors.", message))
 	}(time.Now())
+	return lm.svc.Authorize(ctx, subject, object, relation)
+}
 
-	return lm.svc.Authorize(ctx, token, sub, obj, act)
+func (lm *loggingMiddleware) AddPolicy(ctx context.Context, subject, object, relation string) (err error) {
+	defer func(begin time.Time) {
+		message := fmt.Sprintf("Method add_policy %s#%s@%s took %s to complete", object, relation, subject, time.Since(begin))
```

Comment:
There is no need to translate `(object, relation, subject)` tuple to Keto syntax.